app/views/observations/dwc.eml.erb

```diff
@@ -118,7 +118,7 @@
     <coverage>
       <% if @extent %>
       <geographicCoverage>
-        <geographicDescription>Bounding Box</geographicDescription>
+        <geographicDescription>Custom Boundary</geographicDescription>
```

Comment:
@benkeen can you explain why you made this change?

Reply:
I think I was just conflating two things. I was avoiding the "bounding box" language since a circle is no longer a box, but this appears to be a different creature.

Reply:
Roger. Could you change this back to "Bounding Box" then?